packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -59,17 +62,18 @@ function ChoiceButton({
       fill
       label={
         <Box
+          as='span'
           direction='row'
           fill
           align='center'
         >
           {thumbnailSize !== 'none' && src &&
-            <Media
+            <Image
               alt={`${choiceLabel} media`}
```

Comment:
This text will get added to the button label, so it will say something like 'Wildebeest media Wildebeest.'.

Reply:
Here are the choice button labels for the mock task in the storybook. Easy enough to fix.

<img width="624" alt="The Firefox accessibility inspector open to list the choice menu labels in the storybook." src="https://user-images.githubusercontent.com/59547/206711968-e5061211-1422-4e2b-8440-20ba2e650324.png">

Reply:
The tests must be missing this bug too.

Reply:
Nice catch! I don't have an opinion of which of the two solution.